### ISSUE TITLE
Integrate webview to make standalone mode?

### ISSUE THREAD

**Issue by Pentaphon** [nzbget/android#16]
Would it be possible to have this app act as a standalone app instead of running nzbget and connecting to it via your default browser? The main issue I have with this app is that the nzbget process can easily get killed on an Android phone, and I've been getting around this by running nzbget and my browser in dualscreen mode. However, if there were a way to simply make nzbget for android a standalone app, we wouldn't have these issues and many people who just want to download nzbs to their phone can use the app like they would a dedicated nzb app.

**Comment by hugbug** [comment]
Are you using the most recent version of the app? Version 2.0-testing-r34 has extra code aimed at preventing of the app to be killed.

**Comment by Pentaphon** [comment]
> Are you using the most recent version of the app? Version 2.0-testing-r34 has extra code aimed at preventing of the app to be killed.

Yes, but it's not enough. There needs to be something that keeps nzbget running no matter what, like a notification with a "close nzbget" button that displays while it is running. Or a standalone mode.

**Comment by hugbug** [comment]
The app runs as background process and if I remember correctly **there is** a notification with close button.

I don't have any android device anymore to test this (or to add any other functions).

Maybe someone could develop an improvement? Whenever possible I would prefer a solution not involving integrating of web view.

**Comment by Pentaphon** [comment]
> The app runs as background process and if I remember correctly **there is** a notification with close button.

There is a notification but it doesn't have a close button and from my usage over the past year, it is very easy for the process to be killed despite that notification. It should be much harder to kill the process, using whatever audio players use to keep playing audio while the user is going through tons of apps while listening. Perhaps some kind of keep alive method? I'm not sure. Maybe @kabukky would be interested in revisiting his previous improvements which were a great help, but did not make the nzbget process difficult to kill.
